salt/fileclient.py

```diff
@@ -1356,6 +1356,7 @@ class FSClient(RemoteClient):
     the FSChan object
     '''
     def __init__(self, opts):  # pylint: disable=W0231
+        Client.__init__(self, opts)  # pylint: disable=W0233
```

Comment:
preference here?

Should i just ignore the pylint, or add an option to pass to RemoteClient that doesn't do the extra initialization, but still initializes Client?